nautobot/extras/plugins/__init__.py

```diff
@@ -68,7 +68,8 @@ class PluginConfig(AppConfig):
     datasource_contents = "datasources.datasource_contents"
     graphql_types = "graphql.types.graphql_types"
     jobs = "jobs.jobs"
-    menu_items = "navigation.menu_items"
+    menu_plugin_items = "navigation.menu_items"
```

Comment:
Don't rename this setting - this would be a potentially breaking plugin API change, as an existing plugin might be setting `menu_items` to load its menus from a different module path.